bundles/org.openhab.core/src/main/java/org/openhab/core/internal/items/GroupFunctionHelper.java

```diff
@@ -12,19 +12,25 @@
  */
 package org.openhab.core.internal.items;
 
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 import javax.measure.Quantity;
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
 import org.openhab.core.items.GroupFunction;
+import org.openhab.core.items.Item;
 import org.openhab.core.items.dto.GroupFunctionDTO;
+import org.openhab.core.library.items.NumberItem;
 import org.openhab.core.library.types.ArithmeticGroupFunction;
 import org.openhab.core.library.types.DateTimeGroupFunction;
 import org.openhab.core.library.types.QuantityTypeArithmeticGroupFunction;
 import org.openhab.core.library.types.StringType;
 import org.openhab.core.types.State;
+import org.openhab.core.types.TypeParser;
+import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
```

Comment:
FTR: This class makes use if static logger calls in non-static methods which in general can be replaced. E.g.

https://github.com/cweitkamp/openhab-core/blob/d1ab014173aa254284989a83ae329c70c09fea1b/bundles/org.openhab.core/src/main/java/org/openhab/core/internal/items/GroupFunctionHelper.java#L116-L117